### DISCUSSION TITLE
How can I change the location of. / lotus as the current disk space is insufficient

### DISCUSSION THREAD

**Discussion by Sakuya-CN** [filecoin-project/lotus#6733] [Storage Provider] [answered]
. lotus is in the / root / path. Now it is prompted that the disk space is insufficient. How can I change the location of. / lotus? I don't seem to see any way
![image](https://user-images.githubusercontent.com/56291932/125235300-d4902680-e314-11eb-9911-e5149aeb9b4e.png)
![image](https://user-images.githubusercontent.com/56291932/125235358-ec67aa80-e314-11eb-94ea-0fe5dea6c8a8.png)

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
the env var you are looking for is actually `LOTUS_PATH`. make it available in the daemons environment before staring the daemon

https://docs.filecoin.io/get-started/lotus/send-and-receive-fil/#exporting-and-importing-addresses is what you should do before moving it
